conftest.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
Nit: This shebang is in a lot of our files. As I understand it's only needed if you're going to run the file directly from a terminal. I'd like to remove them whenever they're unnecessary.